targets/TARGET_ONSEMI/TARGET_NCS36510/sleep.c

```diff
@@ -7,11 +7,11 @@
  * $Rev:  0.1 $
  * $Date: 01-21-2016 $
  ******************************************************************************
- * Copyright 2016 Semiconductor Components Industries LLC (d/b/a �ON Semiconductor�).
+ * Copyright 2016 Semiconductor Components Industries LLC (d/b/a �ON Semiconductor�).
```

Comment:
why are these changed? github does not show the characters properly now?

Reply:
Weird, must be some encoding issue with the file or they are not `"` but some weird unicode and my editor went crazy seeing that. I'll revert that.